### PR TITLE
Adding more comparison configs

### PR DESCRIPTION
**Pull request by paigerube14**
Adding in more files that we can gather data in our google sheets with: 

https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/kube-burner-touchstone/7/console

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -311,6 +311,11 @@ pipeline {
                         elif [[ $WORKLOAD == *"node-density"* ]] || [[ $WORKLOAD == "pod-density-heavy" ]]; then
                             export PODS_PER_NODE=$VARIABLE
                         fi
+
+                        if [[ $WORKLOAD == "max-services" ]] || [[ $WORKLOAD == "max-namespaces" ]] || [[ $WORKLOAD == "cluster-density" ]] || [[ $WORKLOAD == "concurrent-builds" ]]; then 
+                            export COMPARISON_CONFIG=$(echo ${COMPARISON_CONFIG/nodeWorkers/nodeAggWorkers})
```
Comment (svetsa-rh) [inline Jenkinsfile:316]: @paigerube14 Jenkins output is showing file path as 
03-28 10:06:44.728  config benchmark-comparison/config/nodeWorkers.json 

Is it supposed to be benchmark-comparison/config/nodeWorkers/nodeAggWorkers.json?

**Comment by paigerube14** [comment]
Passing for cluster-density with removing kublet and crio jsons: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/kube-burner-touchstone/10/console

**Comment by paigerube14** [comment]
FYI I am rewriting the COMPARISON_CONFIG variable because those workloads in the if statement have the metrics-aggregated file as their default and that file doesn't have the certain metrics those files will collect data on 

Able to see workload types here: https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/run.sh

**Pull request merged**